app/workers/education_form/forms/va10203.rb

```diff
@@ -2,8 +2,17 @@
 
 module EducationForm::Forms
   class VA10203 < Base
+    def initialize(app)
+      @stem_automated_decision = app.education_stem_automated_decision
```

Comment:
Should this be defined as a getter?

Reply:
This is overriding the `initialize` method from https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/workers/education_form/forms/base.rb#L61 so that an additional property can be set for 10203 forms during this `Process10203Submission` and during `CreateDailySpoolFiles` sidekiq jobs